README.md

```diff
@@ -524,11 +524,12 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
 
 ## Monitoring and Administration
 
+  * [Dj Diabetes](https://github.com/foxmask/dj-diabetes) - My Glucose Manager - follow your daily health  `BSD` `Python`
   * [Prometheus](http://prometheus.io/) - Monitoring system and time series database. ([Source code](https://github.com/prometheus/prometheus)) `Apache` `Go`
   * [psdash](https://github.com/Jahaja/psdash) - A linux system information web dashboard using psutils and flask
   * [pyDash](https://github.com/k3oni/pydash) - Small web-based monitoring dashboard for linux ([Demo](http://pydash.hostechs.com/main/) - _admin/admin_)
   * [sensu](https://sensuapp.org/) - monitor servers, services, application health, and business KPIs. Get notified about failures before your users do. Collect and analyze custom metrics. `MIT`
-   * [uchiwa](https://uchiwa.io/) - simple dashboard for sensu
+  * [uchiwa](https://uchiwa.io/) - simple dashboard for sensu
```

Comment:
@foxmask please do not remove the indentation here (uchiwa should be indented under sensu)